tools/start_server.py

```diff
@@ -43,16 +43,20 @@
 from http.server import SimpleHTTPRequestHandler
```

Comment:
i accidentally made these whitespace changes, please let me know if I should revert them